### DISCUSSION TITLE
Original photos get deleted though they should not.

### DISCUSSION THREAD

**Discussion by Hyoengju-Johannes-Lee** [photoprism/photoprism#2691] [Q&A]
PLEASE PROCEED ONLY IF YOU ARE ABSOLUTELY SURE THAT THIS IS NOT A TECHNICAL SUPPORT INCIDENT AND/OR POSSIBLY A PROBLEM WITH SOME OTHER SOFTWARE YOU ARE USING. VISIT <https://photoprism.app/kb/getting-support> TO LEARN MORE ABOUT OUR SUPPORT OPTIONS. THANK YOU FOR YOUR CAREFUL CONSIDERATION!

---------------------------------------------------------------------------

#### 1. What is not working as documented?

Please be as specific as possible and explain what part of the software is not [working as documented](https://docs.photoprism.app/). "No photos found" would not be detailed enough. Also, never report [known issues](https://docs.photoprism.app/known-issues/) or [features not yet implemented](https://github.com/photoprism/photoprism/issues) as bugs. Thank you!

#### 2. How can we reproduce it?

1. Have some photos inside a folder. Let's say the folder name is "A".
2. Rename the folder to "B". 
3. Put all photos of the folder "A" (it still remains as A in web before indexing) to archive.
4. Start indexing.
5. After a while, delete the photos that are in the archive.
6. You will see that the 'indexed' photos are deleted in the folder "B" and get angry as I did.

When reporting an import, indexing, or performance issue, please include the number and type of pictures in your library, as well as any configuration options you have changed, such as for thumbnail quality.

#### 3. What behavior do you expect?
No photo from B should be deleted, but just sidecars about the folder "A" should be deleted.

#### 4. What could be the cause of your problem?

Always try to determine the cause of your problem using the checklists at <https://docs.photoprism.app/getting-started/troubleshooting/> before submitting a bug report.

#### 5. Can you provide us with example files for testing, error logs, or screenshots?

Please include sample files or screenshots that help to reproduce your problem. You can also email files or share a download link, see <https://photoprism.app/contact> for details.

Visit <https://docs.photoprism.app/getting-started/troubleshooting/browsers/> to learn how to diagnose frontend issues.

**Important: If it is an import, indexing or metadata issue, we require sample files and logs from you.** Otherwise, we will not be able to process your report. If it is an import problem specifically, please always provide us with an archive of the files before you imported them so we can reproduce the behavior.

#### 6. Which software versions do you use?

(a) PhotoPrism Architecture & Build Number: AMD64, ARM64, ARMv7,...

(b) Database Type & Version: MariaDB, MySQL, SQLite,...

(c) Operating System Types & Versions: Linux, Windows, Android,...

(d) Browser Types & Versions: Firefox, Chrome, Safari on iPhone,...

(e) Ad Blockers, Browser Plugins, and/or Firewall Software?

You can find the version/build number of the app in *Settings* by scrolling to the bottom. Note that MySQL 8 support has been discontinued, see system requirements at <https://docs.photoprism.app/getting-started/#system-requirements>.

*Always provide database and operating system details if it is a backend, import, or indexing issue. Should it be a frontend issue, at a minimum we require you to provide web browser and operating system details. When reporting a performance problem, we ask that you provide us with complete information about your environment, as there may be more than one cause.*

#### 7. On what kind of device is PhotoPrism installed?

This is especially important if you are reporting a performance, import, or indexing issue. You can skip this if you're reporting a problem you found in our public demo, or if it's a completely unrelated issue, such as incorrect page layout.

(a) Device / Processor Type: Raspberry Pi 4, Intel Core i7-3770, AMD Ryzen 7 3800X,...

(b) Physical Memory & Swap Space in GB

(c) Storage Type: HDD, SSD, RAID, USB, Network Storage,...

(d) Anything else that might be helpful to know?

*Always provide device, memory, and storage details if you have a backend, performance, import, or indexing issue.*

#### 8. Do you use a Reverse Proxy, Firewall, VPN, or CDN?

If yes, please specify type and version. You can skip this if you are reporting a completely unrelated issue.

*Always provide this information when you have a reliability, performance, or frontend problem, such as failed uploads, connection errors, broken thumbnails, or video playback issues.*

**Using NGINX?** Please also provide the configuration and/or consider asking the NGINX community for advice as we do not specialize in supporting their product. Docs can be found at <https://docs.photoprism.app/getting-started/proxies/nginx/>.

**Comment by Hyoengju-Johannes-Lee** [comment]
It is not Q&A and a question. It's an issue that you need to change at some point. There should not be a destructive behavior of original photos in such a case.

**Reply by lastzero** [reply]
Please use our bug report template and answer all the questions. I've marked it incomplete as this is currently not the case. Also note we are out of the office and do not have a lot of time to collect all the details personally from you. Thank you very much!!

**Reply by lastzero** [reply]
Also please delete the placeholders and hints.

**Comment by graciousgrey** [comment]
Works as designed as far as I can see.

When renaming something on the filesystem a re-index is needed so that PhotoPrism can read the changes.

PhotoPrism does not create duplicates, when you rename folders. So you don't need to delete photos when renaming folders.

If you move photos to the archive they stay in the archive even if you have moved them to another folder.
If you delete files from the archive it is expected that they are deleted permanently (https://docs.photoprism.app/user-guide/organize/delete/)
